docs/Hosting-DID-Document.md

```diff
@@ -0,0 +1,131 @@
+# Creating a DID Document
```

Comment:
I don't see link in the docker compose readme file to this document

Reply:
I've added a link to this file from the docker compose README.md